app/assets/stylesheets/events.scss

```diff
@@ -360,3 +364,80 @@ input#event_photo_upload{
   font-family: $font-family;
   text-transform: uppercase;
 }
+
+.milestones{
+  opacity: 0.8;
+  .stats-large{
```

Comment:
@andela-megwali Same as above. I will stop mentioning this. Please check for other occurrences and fix accordingly